apps/vr-tests/src/stories/TextConverged.stories.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import Screener from 'screener-storybook/src/screener';
+import {
+  Body,
+  Caption,
+  Display,
+  Headline,
+  LargeTitle,
+  Subheadline,
+  Text,
+  Title1,
+  Title2,
+  Title3,
+} from '@fluentui/react-text';
+import { FluentProviderDecorator } from '../utilities/index';
+
+storiesOf('Text Converged', module)
```

Comment:
```suggestion
storiesOf('react-text Text', module)
```

Can we prefix stories with package name like in `react-button`?
https://github.com/microsoft/fluentui/blob/147ad1114e2e9261aae0908b98b7244a8b8997a7/apps/vr-tests/src/stories/ReactButton.stories.tsx#L12

It's not a defined convention, but IMO in this case difference between v8 is more visible.